ote_cli/ote_cli/tools/deploy.py

```diff
@@ -55,6 +56,12 @@ def main():
     # Parses input arguments.
     args = parse_args()
 
+    # Validate required paths that are sourced in args
+    validate_path(args.load_weights)
+
+    if args.save_model_to:
+        validate_path(args.save_model_to)
```

Comment:
could you try to do something more elegant ?
https://stackoverflow.com/questions/37471636/python-argument-parsing-validation-best-practices
https://stackoverflow.com/questions/11540854/file-as-command-line-argument-for-argparse-error-message-if-argument-is-not-va
...